core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -101,23 +141,124 @@ private void initializeCamera() {
         fontCamera.setToOrtho(false, APPLICATION_WIDTH, APPLICATION_HEIGHT);
     }
 
+    public void drawMatchScreen(){
```

Comment:
Das können wir vielleicht in mehrere Methoden aufteilen
+JavaDoc

Reply:
Aufgeteilt in mehrere Methoden und JavaDoc hinzugefügt.